### PR TITLE
Prl 6391

### PR DESCRIPTION
**Pull request by Scralfie**
### Other proceedings 6

List of pages:

otherInjuctionOrder/order-details
otherInjuctionOrder/1/documentUpload
undertakingOrder/order-details
undertakingOrder/1/documentUpload
otherOrder/order-details
otherOrder/1/documentUpload
User(s) to complete event: Citizen

Definition of done:

All POM sections are created
All POMs are added to a relevant journey

### JIRA link

[PRL-6391](https://tools.hmcts.net/jira/browse/PRL-6391)

**Before merging a pull request make sure that:**

- [x] Commits are meaningful and simple
- [x] All commits are squashed into a single commit
- [x] README and other documentation has been updated / added (if needed)

### CODE REVIEW

File: e2e/journeys/citizen/createCase/C100OtherProceedings/C100OtherProceedings.ts [hunk 1]
```diff
@@ -66,6 +75,9 @@ export class C100OtherProceedings {
     yesNoOccupationOrderDetails,
     yesNoForcedMarriageProtectionOrderDetails,
     yesNoRestrainingOrderDetails,
+    yesNoOtherInjunctionOrderDetails,
+    yesNoUndertakingOrderDetailsOrderDetails,
+    yesNoOtherOrderDetailsDetails,
```
Comment (OLT2000) [inline e2e/journeys/citizen/createCase/C100OtherProceedings/C100OtherProceedings.ts:80]: Can you make a single argument for the whole subJourney: `c100OtherProceedingsYesNoToAll` so it simplifies the number of variables? We're only going to be doing yes to all or no to all in the tests.

**Review by OLT2000 — CHANGES_REQUESTED** [review]

File: e2e/journeys/citizen/createCase/C100OtherProceedings/C100OtherProceedings.ts (hunk 1 above)
Comment (Scralfie) [reply e2e/journeys/citizen/createCase/C100OtherProceedings/C100OtherProceedings.ts:80]: Was planning to do that in a different PR

Comment (OLT2000) [reply e2e/journeys/citizen/createCase/C100OtherProceedings/C100OtherProceedings.ts:80]: Is this the last OP ticket? I'm happy to make this change in the stitching effort if so

Comment (Scralfie) [reply e2e/journeys/citizen/createCase/C100OtherProceedings/C100OtherProceedings.ts:80]: No, I have to do the summary page for it. I was planning on doing it in that ticket

**Review by JakeCG — APPROVED** [review]

**Review by OLT2000 — APPROVED** [review]

**Pull request merged**